src/renderer/features/operations/OperationsValidation/lib/transfer-rules.ts

```diff
@@ -259,10 +259,17 @@ export const transferValidator = createTxValidator<{
       const isXcm = destinationChain.chainId !== sourceChain.chainId;
       const totalAmount = isXcm && destinationFee && !destinationFee.isZero() ? amount.add(destinationFee) : amount;
       const preservation = isXcm ? 'keepAlive' : balancePreservation;
+      const withdrawalResult = balanceService.tryWithdraw(balance, totalAmount, preservation);
 
       return {
         account: initiator,
-        balance: balanceService.tryWithdraw(balance, totalAmount, preservation),
+        balance:
+          isXcm && destinationFee && !destinationFee.isZero()
+            ? {
+                ...withdrawalResult,
+                required: amount,
+              }
```

Comment:
we should think twice if we missed any scenario with this withdrawal result editing

and why the `balanceService.tryWithdraw(balance, amount, preservation)` would be incorrect?

Reply:
@johnthecat this is the place I was talking about